servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -69,11 +70,18 @@
   private final ExecutorService mFixedExecutionService =
       Executors.newFixedThreadPool(DEFAULT_BLOCK_REMOVER_POOL_SIZE);
 
-  BlockMasterSync(BlockDataManager blockDataManager, TachyonConf tachyonConf,
-      NetAddress workerAddress, WorkerBlockMasterClient masterClient) {
+  /**
+   * Constructor for BlockMasterSync
+   *
+   * @param blockDataManager the blockDataManager this syncer is updating to
+   * @param workerAddress the net address of the worker
+   * @param masterClient the Tachyon master client
+   */
+  BlockMasterSync(BlockDataManager blockDataManager, NetAddress workerAddress,
+      WorkerBlockMasterClient masterClient) {
     mBlockDataManager = blockDataManager;
     mWorkerAddress = workerAddress;
-    mTachyonConf = tachyonConf;
+    mTachyonConf = WorkerContext.getConf();
```

Comment:
Can we get rid of mTachyonConf and instead use WorkerContext.getCon() where it is used?